example/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "example",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@etherspot/etherspot-ui": "file:./../",
```

Remember to add instructions in the README in regards to running rollup first otherwise this won't work. Also shouldn't this be:

```suggestion
    "@etherspot/etherspot-ui": "file:./../dist",
```
or something?